cdk/apps/dev_worker/app.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+
+from aws_cdk import core
+
+from dev_worker.dev_worker_stack import DevWorkerStack
+
+# Set account vars
+ACCOUNT = "843407916570"  # Dev
+REGION = "ap-southeast-2"
+CREATOR_NAME = "Alexis Lucattini"
+
+# Set tags for container
+TAGS = {
+    # Name is set to gridss-purple-lynx
+    "Stack": "DEV-CDK",
+    "UseCase": "Testing-CDK",
+    "Creator": "Alexis Lucattini",
```

Comment:
read from context?

Reply:
Or env (i.e. $USER)?

Reply:
My issue with context is that I can't nest attributes. Or should these be separate inputs, rather than under the nest - tags.